$MirOS: ports/www/lynx/patches/patch-lynx_man,v 1.4 2010/12/23 20:25:31 tg Exp $

	site policy; some fix spelling

--- lynx.man.orig	Mon Feb 13 02:45:53 2012
+++ lynx.man	Sat Mar  3 18:20:22 2012
@@ -218,16 +218,16 @@ it is equivalent to setting the \fBCOLOR
 library like \fBncurses\fR, \fILynx\fP relies completely on the terminal
 description to determine whether color mode is possible, and
 this flag is not needed and thus unavailable.)
-A saved show_color=always setting found in a .lynxrc file at
+A saved show_color=always setting found in a .etc/lynxrc file at
 startup has the same effect.  A saved show_color=never found
-in .lynxrc on startup is overridden by this flag.
+in .etc/lynxrc on startup is overridden by this flag.
 .TP
 .B \-connect_timeout\fR=\fIN
 Sets the connection timeout, where N is given in seconds.
 .TP
 .B \-cookie_file\fR=\fIFILENAME
 specifies a file to use to read cookies.
-If none is specified, the default value is ~/.lynx_cookies
+If none is specified, the default value is ~/.etc/cookies.txt
 for most systems, but ~/cookies for MS-DOS.
 .TP
 .B \-cookie_save_file\fR=\fIFILENAME
@@ -420,7 +420,7 @@ within a mailto URL or form ACTION.
 .TP
 .B \-nocolor
 force color mode off, overriding terminal capabilities and any
--color flags, COLORTERM variable, and saved .lynxrc settings.
+-color flags, COLORTERM variable, and saved .etc/lynxrc settings.
 .TP
 .B \-noexec
 disable local program execution. (DEFAULT)
@@ -638,7 +638,7 @@ disallow mail.
 disallow USENET News posting.
 .TP
 .B options_save
-disallow saving options in .lynxrc.
+disallow saving options in .etc/lynxrc.
 .TP
 .B outside_ftp
 disallow ftps for people coming from outside your
@@ -663,13 +663,13 @@ outside your domain (utmp required for s
 disallow most print options.
 .TP
 .B shell
-disallow shell escapes and lynxexec or lynxprog G)oto's.
+disallow shell escapes and lynxexec or lynxprog G)oto.
 .TP
 .B suspend
 disallow Unix Control-Z suspends with escape to shell.
 .TP
 .B telnet_port
-disallow specifying a port in telnet G)oto's.
+disallow specifying a port in telnet G)oto.
 .TP
 .B useragent
 disallow modifications of the User-Agent header.
@@ -737,7 +737,7 @@ lynx -source . >foo.html
 .IP
 generates HTML source listing the files in the current directory.
 Each file is marked by an HREF relative to the parent directory.
-Add a trailing slash to make the HREF's relative to the current directory:
+Add a trailing slash to make the HREFs relative to the current directory:
 .sp
 .RS
 lynx -source ./ >foo.html
@@ -807,7 +807,7 @@ color-style
 TRST (table layout)
 .TP
 .B 8
-configuration (lynx.cfg, .lynxrc, .lynx-keymaps, mime.types and mailcap contents)
+configuration (lynx.cfg, .etc/lynxrc, .lynx-keymaps, mime.types and mailcap contents)
 .TP
 .B 16
 binary string copy/append, used in form data construction.
@@ -973,7 +973,7 @@ the contents of an /etc/organization fil
 .TP
 .B NNTPSERVER
 If set, this variable specifies the default NNTP server that will be used
-for \fBUSENET\fR news reading and posting with \fILynx\fR, via news: URL's.
+for \fBUSENET\fR news reading and posting with \fILynx\fR, via news: URLs.
 .TP
 .B ORGANIZATION
 This variable, if set, provides the string used in the \fBOrganization:\fR
@@ -1201,7 +1201,7 @@ and the WWW community.  Also a special t
 much of \fILynx\fP to VMS and did or organized most of its development since the
 departures of Lou Montulli and Garrett Blythe from the University of Kansas
 in the summer of 1994 through the release of v2.7.2, and to everyone
-on the net who has contributed to \fILynx\fP's development either directly (through
+on the net who has contributed to \fILynx\fP' development either directly (through
 patches, comments or bug reports) or indirectly (through inspiration and
 development of other systems).
 .SH AUTHORS
